### PR TITLE
mgr/dashboard: UI fixes

### PR DESCRIPTION
**Pull request by avanthakkar**
Fixes: https://tracker.ceph.com/issues/42914

Signed-off-by: Avan Thakkar <athakkar@redhat.com>

## Checklist
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: monitoring/grafana/dashboards/pool-detail.json [hunk 1]
```diff
@@ -257,7 +257,7 @@
       "thresholds": [],
       "timeFrom": null,
       "timeShift": null,
-      "title": "$pool_name Object In-/Egress",
+      "title": "$pool_name Object In-Progress",
```
Comment (epuertat) [inline monitoring/grafana/dashboards/pool-detail.json]: I think this means `Ingress` (input data traffic) + `Egress` (output data traffic).

File: src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.ts [hunk 2]
```diff
@@ -261,7 +261,7 @@ export class OsdListComponent implements OnInit {
       },
       {
         prop: 'stats_history.in_bytes',
-        name: this.i18n('Writes bytes'),
+        name: this.i18n('Write bytes'),
```
Comment (epuertat) [inline src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.ts]: These two columns refer to the number of bytes read and written, thus I guess it should be `Bytes read` and `Bytes written`.

File: src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/prometheus/silence-matcher-modal/silence-matcher-modal.component.html [hunk 3]
```diff
@@ -73,7 +73,7 @@
                  id="is-regex">
           <label for="is-regex"
                  class="custom-control-label"
-                 i18n>Use regular expression</label>
+                 i18n>Enable</label>
```
Comment (epuertat) [inline src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/prometheus/silence-matcher-modal/silence-matcher-modal.component.html]: Why is this change?

**Comment by TimothyAsirJeyasing** [comment]
/proc/<procid>/io returns how many IO bytes a process has read or write.
It provides details like:
read_bytes: 25780224
write_bytes: 0
cancelled_write_bytes: 0

File: monitoring/grafana/dashboards/pool-detail.json (hunk 1 above)
Comment (avanthakkar) [reply monitoring/grafana/dashboards/pool-detail.json]: Yes. I have changed it to "Object Ingress/Egress"

File: src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/prometheus/silence-matcher-modal/silence-matcher-modal.component.html (hunk 3 above)
Comment (avanthakkar) [reply src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/prometheus/silence-matcher-modal/silence-matcher-modal.component.html]: Yes, "Use regular expression" makes more sense. I have changed it back to it.

**Comment by a2batic** [comment]
jenkins retest this please

**Comment by alfonsomthd** [comment]
jenkins test dashboard

**Comment by avanthakkar** [comment]
jenkins test dashboard

**Pull request merged**